apps/developer-hub/src/components/HomepageHero/HeroContent/index.module.scss

```diff
@@ -0,0 +1,59 @@
+@use "@pythnetwork/component-library/theme";
+
+.heroContent {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  @include theme.breakpoint(md) {
+    grid-column: 1 / 2;
+  }
+
+  @include theme.breakpoint(lg) {
+    grid-column: 1 / 3;
+  }
+}
+
+.title {
+  font-size: theme.spacing(6);
```

Comment:
It would be better to stick to font sizes from the [font-size scale](https://github.com/pyth-network/pyth-crosschain/blob/main/packages/component-library/src/theme.scss#L33-L48) -- I have a [text mixin](https://github.com/pyth-network/pyth-crosschain/blob/main/packages/component-library/src/theme.scss#L874-L880) that I usually apply e.g. `@include theme.text("sm", "medium")`.

TBH I'm not in love with the way the font configuration stuff is done, but at the very least I think we should try to stick to font sizes out of the font-size scale one way or another.